tutorials/20_REST_API.ipynb

```diff
@@ -0,0 +1,538 @@
+    "\n",
+    "```yaml\n",
+    "pipelines:\n",
+    "  ...\n",
+    "  - name: indexing\n",
+    "    nodes:\n",
+    "      - name: FileTypeClassifier\n",
+    "        inputs: [File]\n",
+    "      - name: TextFileConverter\n",
+    "        inputs: [FileTypeClassifier.output_1]\n",
+    "      - name: PDFFileConverter\n",
+    "        inputs: [FileTypeClassifier.output_2]\n",
+    "      - name: Preprocessor\n",
+    "        inputs: [PDFFileConverter, TextFileConverter]\n",
+    "      - name: Retriever\n",
+    "        inputs: [Preprocessor]\n",
+    "      - name: DocumentStore\n",
+    "        inputs: [Retriever]\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After completing query and indexing pipelines, add `version: ignore` to the top of the file. Now, the pipeline YAML is completed.\n",
+    "\n",
+    "```yaml\n",
+    "version: ignore\n",
+    "\n",
+    "components:\n",
+    "  - name: DocumentStore\n",
+    "    type: ElasticsearchDocumentStore\n",
+    "  - name: Retriever\n",
+    "    type: EmbeddingRetriever\n",
+    "    params:\n",
+    "      document_store: DocumentStore\n",
+    "      top_k: 5 \n",
+    "      embedding_model: sentence-transformers/multi-qa-mpnet-base-dot-v1\n",
+    "  - name: FileTypeClassifier\n",
+    "    type: FileTypeClassifier\n",
+    "  - name: TextFileConverter\n",
+    "    type: TextConverter\n",
+    "  - name: PDFFileConverter\n",
+    "    type: PDFToTextConverter\n",
+    "  - name: Preprocessor\n",
+    "    type: PreProcessor\n",
+    "    params:\n",
+    "      split_by: word\n",
+    "      split_length: 1000\n",
+    "      split_overlap: 50 \n",
+    "      split_respect_sentence_boundary: True\n",
+    "\n",
+    "pipelines:\n",
+    "  - name: query \n",
+    "    nodes:\n",
+    "      - name: Retriever\n",
+    "        inputs: [Query]\n",
+    "  - name: indexing\n",
+    "    nodes:\n",
+    "      - name: FileTypeClassifier\n",
+    "        inputs: [File]\n",
+    "      - name: TextFileConverter\n",
+    "        inputs: [FileTypeClassifier.output_1]\n",
+    "      - name: PDFFileConverter\n",
+    "        inputs: [FileTypeClassifier.output_2]\n",
+    "      - name: Preprocessor\n",
+    "        inputs: [PDFFileConverter, TextFileConverter]\n",
+    "      - name: Retriever\n",
+    "        inputs: [Preprocessor]\n",
+    "      - name: DocumentStore\n",
+    "        inputs: [Retriever]\n",
+    "```\n",
+    "\n",
+    "Feel free to play with the pipeline setup later on. Add or remove some nodes, change the parameters or add new ones. Check out [Haystack API Reference](https://docs.haystack.deepset.ai/reference/answer-generator-api) for more options on nodes and parameters."
```

Comment:
```suggestion
    "Feel free to play with the pipeline setup later on. Add or remove some nodes, change the parameters, or add new ones. For more options for nodes and parameters, check out [Haystack API Reference](https://docs.haystack.deepset.ai/reference/answer-generator-api)."
```

With links, we always specify the purpose first, so " To learn about blah, go to blah" :)